### PR TITLE
Update configure-helper.sh to support heapster resource optimizations

### PR DESCRIPTION
**Pull request by vantuvt**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -2348,10 +2348,18 @@ EOF
...
     nanny_memory="90Mi"
-    local -r metrics_memory_per_node="${HEAPSTER_GCP_MEMORY_PER_NODE:-4}"
+    local heapster_min_cluster_size="16"
+    local metrics_memory_per_node="${HEAPSTER_GCP_MEMORY_PER_NODE:-4}"
     local -r metrics_cpu_per_node="${HEAPSTER_GCP_CPU_PER_NODE:-0.5}"
     local -r eventer_memory_per_node="500"
     local -r nanny_memory_per_node="200"
+    if [[ "${ENABLE_SYSTEM_ADDON_RESOURCE_OPTIMIZATIONS:-}" == "true" ]]; then
+      # TODO: Set each of the following variables to optimized values.
+      base_metrics_memory="140Mi"
```
Comment (gmarek) [inline cluster/gce/gci/configure-helper.sh]: You probably wanted "${HEAPSTER_GCP_BASE_MEMORY:-140Mi}". I'd drop this 'if' as it's a no-op. It's easy enough to add when you'll have the proper logic figured out.

Comment (vantuvt) [reply cluster/gce/gci/configure-helper.sh]: Good catch! Thanks. This logic has been implemented to facilitate testing of a complementary change that sets the ENABLE_SYSTEM_ADDON_RESOURCE_OPTIMIZATIONS environment variable.

**Comment by vantuvt** [comment]
I've updated this PR with the final scaling policy numbers. So the TODO I had originally added has now been cleaned up. PTAL. Thanks!

**Pull request merged**